Makefile

```diff
@@ -12,7 +12,7 @@ package:
 	-v ${PWD}:/go/src/github.com/CMSgov/bcda-ssas-app packaging $(version)
 
 lint:
-	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint --deadline=3m run ./...
+	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint --deadline=3m -e SA1029 run ./...
```

Comment:
I'm curious why this is disabled in this repo and not in the[ associated ACO API PR](https://github.com/CMSgov/bcda-app/pull/465).

Reply:
The technique that was performed in the ACO API PR was reproduced in SSAS, and works fine... however, the is some logging in SSAS which does not play well with ContextKey.  I could not get it to work.  Maybe we should write a tech debt ticket to address it later?

Reply:
Reference: https://travis-ci.org/github/CMSgov/bcda-ssas-app/builds/666583174